Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -524,6 +524,18 @@ public void deleteConcept(String id, String userName) throws LuceneException, In
         indexService.deleteById(concept.getId(), userName);
     }
 
+    public void deleteNonMergedConcept(String id, String userName) throws LuceneException, IndexerRunningException {
+        ConceptEntry concept = client.getEntry(id);
+        concept.setDeleted(true);
+        ChangeEvent changeEvent = new ChangeEvent();
+        changeEvent.setType(ChangeEventTypes.DELETION);
+        changeEvent.setDate(new Date());
+        changeEvent.setUserName(userName);
+        concept.addNewChangeEvent(changeEvent);
+        client.update(concept, DBNames.DICTIONARY_DB);
+        indexService.deleteById(concept.getId(), userName);
```

Comment:
the previous method does pretty much exactly the same thing. just have one method and give it the concept to delete.